### ISSUE TITLE
Difference between article and source code

### ISSUE THREAD

**Issue by RicWilliamsCC** [apirker/eShoppingSimple#2]
The Article reference  "publishes the event OrderCompletedEvent after updating the repository, which makes sure that other services get the information about the new order." The sample code does not include the OrderCompletedEvent. Is there some additional description on that specific change?

**Comment by apirker** [comment]
That is true, I believe i forgot to put this event implementation in the corresdonding assembly of the events. Would be rather straightforward to implement by just implementing the IEvent interface, since any publishable event needs to implement this interface.
